libs/execution/src/lib/types/io-types/filesystem-inmemory.ts

```diff
@@ -51,8 +51,12 @@ export class InMemoryFileSystem implements FileSystem {
     return null;
   }
 
+  getPathSeparator(): string {
+    return InMemoryFileSystem.PATH_SEPARATOR;
+  }
+
   private processPath(path: string): string[] | null {
-    if (!path.startsWith('/')) {
+    if (!path.startsWith(InMemoryFileSystem.PATH_SEPARATOR)) {
```

Comment:
If you add a getter function you must use it consistently. You can make `PATH_SEPARATOR` private and `getPathSeparator` `static`. And then use the getter function everywhere.

Reply:
made getter static and now in consistent use.